tests/input/obo-compliance/prefixes-conflict-main-idspace/prefixes-conflict-main-idspace.meta.yaml

```diff
@@ -0,0 +1,4 @@
+name: prefixes-conflict-main-idspace
+name: prefixes-conflict-main-idspace
```

Comment:
Why do the metadata yaml files have the same line twice? All of them.

Reply:
I believe this is a bug in the `split_compiled_obo` function (`src/tests/test_converters/test_obo_format.py`), [lines 113–119](https://github.com/INCATools/ontology-access-kit/blob/377f5ac66ab95c72a2700b321c2ac95b39c17829/tests/test_converters/test_obo_format.py#L113). The `name: ...` line is added to the `yamls[ontology_id]` dictionary twice:

```python
if k == "name":
    ontology_id = v
    if ontology_id in blocks:
        raise ValueError(f"Duplicate ontology id: {ontology_id}")
    metadata[ontology_id] = {}
    yamls[ontology_id] = [yaml.rstrip()] # First addition of `name: ...`
    blocks[ontology_id] = [
        "format-version: 1.4\n",
        f"ontology: {ontology_id}\n",
    ]
metadata[ontology_id][k.strip()] = v.strip()
yamls[ontology_id].append(yaml.strip()) # Second addition of `name: ...`
```